apps/video/index.html

```diff
@@ -65,59 +65,73 @@ <h1 id="thumbnails-number-selected"></h1>
         <!-- It gets styled differently depending on what it is over -->
-        <ul id="thumbnails"></ul> <!-- Thumbnails inserted here -->
+        <ul role="presentation" id="thumbnails"></ul> <!-- Thumbnails inserted here -->
 
         <footer id="thumbnails-bottom" class="thumbnails-list">
-          <a id="thumbnails-video-button" class="button" data-icon="video"></a>
-          <a id="thumbnails-select-button" class="button" data-icon="select"></a>
+          <a id="thumbnails-video-button" class="button" data-icon="video"
+             role="button" data-l10n-id="video-button"></a>
+          <a id="thumbnails-select-button" class="button" data-icon="select"
+             role="button" data-l10n-id="select-button"></a>
         </footer>
         <footer id="thumbnails-select-bottom" class="thumbnails-select">
-          <a id="thumbnails-delete-button" class="button" data-icon="delete"></a>
-          <a id="thumbnails-share-button" class="button" data-icon="share"></a>
+          <a id="thumbnails-delete-button" class="button" data-icon="delete"
+             role="button" data-l10n-id="delete-button"></a>
+          <a id="thumbnails-share-button" class="button" data-icon="share"
+             role="button" data-l10n-id="share-button"></a>
         </footer>
       </section>
 
       <!-- (maximized) video player -->
       <section role="region" id="player-view">
         <!-- display this at startup while we create thumbnails -->
-        <div id="spinner-overlay" class="hidden"><div id="spinner"></div></div>
+        <div id="spinner-overlay" class="hidden" role="progress"
+             data-l10n-id="spinner-overlay"><div id="spinner"></div></div>
         <div id="video-container">
-          <video src="about:blank" id="player"></video>
+          <video src="about:blank" id="player" aria-hidden="true"></video>
         </div>
         <!-- video controls -->
         <div id="videoControls">
           <section role="region">
             <gaia-header id="player-header" action="back">
               <h1 id="video-title"></h1>
-              <button id="options" data-icon="more"></button>
+              <button id="options" data-icon="more"
+                      data-l10n-id="options-more"></button>
               <button id="picker-done" data-l10n-id="done">Done</button>
             </gaia-header>
-            <footer id="videoBar">
-              <div id="timeSlider">
-                <span id="elapsed-text"></span>
-                <div id="slider-wrapper">
-                  <div id="elapsedTime" class="progress"></div>
-                  <div id="bufferedTime" class="progress"></div>
-                  <div id="timeBackground" class="progress"></div>
-                  <button id="playHead"></button>
+            <div id="hide-controls" role="button"
+                 data-l10n-id="hide-controls-button"></div>
+            <div role="toolbar">
+              <footer id="videoBar" role="presentation">
```

Comment:
I think it would be helpful to include a comment regarding the "presentation" role and why it's being used here.